### ISSUE TITLE
Widget support

### ISSUE THREAD

**Issue by dtantono** [mastodon/mastodon-android#511]
Hi
I think it will be wonderful if mastodon support widget. What do you think?

**Comment by aCID-sLAM** [comment]
Yeah. Seems like there is no Client until today, that has a widget.

**Comment by MKCOOL142** [comment]
Especially since iOS has widgets added in the last update

**Comment by MKCOOL142** [comment]
I tried to make some concepts in the style that google uses to have more consistency 
![image](https://user-images.githubusercontent.com/40364718/219877959-0002d813-423f-4978-b07e-e17570c2882c.png)
![image](https://user-images.githubusercontent.com/40364718/219877963-08b50ecb-2688-4b92-a3e3-9aeaf2f0cb37.png)
![image](https://user-images.githubusercontent.com/40364718/219877965-8efb545e-22e7-49c5-86f7-1f24d145a146.png)
![image](https://user-images.githubusercontent.com/40364718/219877972-e80a8f21-29ae-4e1a-b7a2-93e10b6c57ee.png)

**Comment by AlphaDE** [comment]
Such an option would be great to get a short glance over the latest toots on the home screen without the need to always open the app.
